mnamer/__init__.py

```diff
@@ -79,17 +80,19 @@
     "api_key_omdb": "61652c15",
     # Non-CLI preferences
     "replacements": {"&": "and", "@": "at", ":": ",", ";": ","},
+    "season_overrides": {}
 }
 
 PREFERENCE_KEYS = set(PREFERENCE_DEFAULTS.keys())
 DIRECTIVE_KEYS = {
     "help",
     "config_dump",
+    "config",
     "config_ignore",
     "id",
     "media_override",
     "media_mask",
     "test",
-    "version",
+    "version"
```

Comment:
This project uses [black](https://github.com/psf/black) for code formatting which places a trailing comma at the end of multi-line expressions. AFAIK the rationale is to minimize diffs when appending to a list, e.g. if we later wanted to add a key after version it would require changing this line again.